uiowa/uiowa
# Transition v3 article blocks to views-based solution

Resolves https://github.com/uiowa/uiowa/issues/3501
Resolves https://github.com/uiowa/uiowa/issues/3272
Resolves #3678
Resolves https://github.com/uiowa/uiowa/issues/4357 




# How to test



- [x] `blt drupal:install --site default` to see the new front page. no default content issues, articles list block is on the front page.
  - Approve new baseline for Percy
- [x] `blt dsa` a lot of sites to test the update hook against. no update/import issues.
- [x] compare existing articles block with the post-update replacement views blocks
  - hide images, descriptions, details should carry over. 
- [x] create a new articles views block, configure it how you will, get the expected result even when paging.
  - All the expected styles a available and working
  - More link goes to SiteNow Articles view page, custom path works if provided.
  - all styles should be "default" to start
  - changing image type, makes size options toggle
- [x] see some that fields located on the views config are not available to editors to toggle visibility
- [x] direct from source link functionality works even when source link is hidden.
- [x] organization is wrapped in source link if source link has value and source link is hidden.
- [x] source link is hidden by default for all new and existing
- [x] check to see if replicate is working for the new articles block

## ToDo
- [x] Investigate https://github.com/uiowa/uiowa/pull/4286#issuecomment-948901829
- [x] Update splits to make sure new block is available and old block is not.
- [x] Investigate empty headlines
- [x] WSOD https://github.com/uiowa/uiowa/pull/4286#issuecomment-949009898

# Additional information

- Due to the update hook, this should be run "off-hours," so merging/deployment should be coordinated.
- Website documentation needs to be updated.
